api/classes/publisher.js

```diff
@@ -310,6 +318,23 @@ class BasePublisher {
     return publishedFilesQueryBuilder
     .getAllModifiedFiles(this.publishedProject.id)
     .then(function(publishedFiles) {
+      if(!updateHtmlFilesWithNewRemixData) {
+        return publishedFiles;
+      }
+
+      var ignorePaths;
```

Comment:
given that this value gets passed into `getAllFilesMatchPath` a few lines below, what should its default value be?